configs/config/pretrain/byol/byol_1gpu.yaml

```diff
@@ -0,0 +1,121 @@
+# @package _global_
+config:
+  MAX_ITER: 312785 # int(1000. * Training set length / 4096) , here set according to ImageNet ILSVRC-2012 dataset
+  WARMUP_ITERS: 3128 # int(10. * Training set length / 4096) , here set according to ImageNet ILSVRC-2012 dataset
+  VERBOSE: False
+  LOG_FREQUENCY: 10
+  TEST_ONLY: False
+  TEST_MODEL: False
+  SEED_VALUE: 1337
+  MULTI_PROCESSING_METHOD: forkserver
+  HOOKS:
+    PERF_STATS:
+      MONITOR_PERF_STATS: True
+      ROLLING_BTIME_FREQ: 313
+    TENSORBOARD_SETUP:
+        USE_TENSORBOARD: True
+        EXPERIMENT_LOG_DIR: "byol_reference"
+        LOG_PARAMS: False
+        FLUSH_EVERY_N_MIN: 20
+  DATA:
+    NUM_DATALOADER_WORKERS: 5
+    TRAIN:
+      DATA_SOURCES: [disk_folder]
+      DATASET_NAMES: [imagenet1k_folder]
+      BATCHSIZE_PER_REPLICA: 64
+      LABEL_TYPE: sample_index    # just an implementation detail. Label isn't used
+      TRANSFORMS:
+        - name: ImgReplicatePil
+          num_times: 2
+        - name: RandomResizedCrop
+          size: 224
+        - name: RandomHorizontalFlip
+          p: 0.5
+        - name: ImgPilColorDistortion
+          strength: 0.5
+        - name: ImgPilMultiCropRandomApply
+          transforms:
+            - name: ImgPilGaussianBlur
+              p: 0.5
+              radius_min: 0.1
+              radius_max: 2.0
+          prob: [ 1.0, 0.1 ]
+        - name: ImgPilMultiCropRandomApply
+          transforms:
+            - name: ImgPilRandomSolarize
+              p: 0.2
+          prob: [ 0.0, 0.2 ]
+        - name: ToTensor
+        - name: Normalize
+          mean: [ 0.485, 0.456, 0.406 ]
+          std: [ 0.229, 0.224, 0.225 ]
+      COLLATE_FUNCTION: simclr_collator
+      MMAP_MODE: True
+      COPY_TO_LOCAL_DISK: False
+      COPY_DESTINATION_DIR: /tmp/imagenet1k/
+      DROP_LAST: True
+  TRAINER:
+    TRAIN_STEP_NAME: standard_train_step
+  METERS:
+    name: ""
+  MODEL:
+    TRUNK:
+      NAME: resnet
+      RESNETS:
+        DEPTH: 50
+        ZERO_INIT_RESIDUAL: True
+    HEAD:
+      PARAMS: [
+        ["mlp", {"dims": [2048, 4096], "use_relu": True, "use_bn": True}],
+        ["mlp", {"dims": [4096, 256]}],
+        ["mlp", {"dims": [256, 4096], "use_relu": True, "use_bn": True}],
+        ["mlp", {"dims": [4096, 256]}],
+      ]
+    SYNC_BN_CONFIG:
+      CONVERT_BN_TO_SYNC_BN: True
+      SYNC_BN_TYPE: pytorch
+    AMP_PARAMS:
+      USE_AMP: False
+  LOSS:
+      name: byol_loss
+      byol_loss:
+        embedding_dim: 256
+        momentum: 0.999
+  OPTIMIZER:
+      name: lars
+      trust_coefficient: 0.001
+      weight_decay: 1.5e-6
+      momentum: 0.9
+      nesterov: False
+      num_epochs: 300
+      regularize_bn: False
```

Comment:
Let's stick to the defaults here: 
```
    regularize_bn: False
    regularize_bias: True
```


Otherwise, bias will not have weight decay.